src/main/resources/application.properties

```diff
@@ -19,6 +19,18 @@ hocs.search-service=http://localhost:8088
 hocs.document-service=http://localhost:8083
 hocs.basicauth=UNSET
 
+bank-holidays-api-url=https://www.gov.uk/bank-holidays.json
+
+notify.queue.name=notify-queue
```

Comment:
I think there is a bit of merge issue here down to line 33

Reply:
I've removed this block